core/domain/classifier_services.py

```diff
@@ -329,15 +337,17 @@ def save_classifier_training_job(algorithm_id, exp_id, exp_version,
     Returns:
         job_id: str. ID of the classifier training job.
     """
-    classifier_training_job_model = (
-        classifier_models.ClassifierTrainingJobModel.get(job_id, False))
-    if classifier_training_job_model is None:
+    if status is None:
+        status = feconf.TRAINING_JOB_STATUS_NEW
```

Comment:
@seanlip I think its  better off this way. For a person creating a job for the first time, we'd need to explicitly assign status values. This is not very good, I believe. While updating though, we have access to the job's previous status, so we can pass it along the function. Do you think so? If not, we can always remove the default args and pass the status as a parameter in all tests.